packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -46,6 +47,7 @@ import {
 } from './template-actions';
 import usePatternSettings from '../page-patterns/use-pattern-settings';
 import { unlock } from '../../lock-unlock';
+import SideEditor from '../page-pages/side-editor';
```

Comment:
It's a bit weird that the "templates" page is importing from the "pages" page.

Reply:
What's a good place to hove the `SideEditor` component so both pages can reuse it?

Reply:
I think it should just be in its own folder under `components`. Maybe `<PostPreview postId={postId} postType={postType} />`. That component shouldn't be an "editor". It's just a preview of the current selected entity.


To be honest, that component as written right now is a hack. The fact that it works is just random. For instance, it's impossible to render two of these components on a page. I'd add a comment to the component to explicitly mention that it's hacky and that it shouldn't be relied on as is forever.

Reply:
Though, we also have `import usePatternSettings from '../page-patterns/use-pattern-settings';` that could find a better place as well.

Reply:
Extracted `SideEditor` to its own `PostPreview` component.